src/app/components/publiclists/challengelist/challengecard/challengecard.component.html

```diff
@@ -1,55 +1,29 @@
-<div class="challenge-card-container">
-  <div class="challenge-img-div" (click)="challengeClicked()">
-    <span class="helper"></span>
-    <div class="challenge-img-parent">
-        <img class="challenge-img" [src]="challenge['image']" (error)="challenge['image']='assets/images/404.png';" />
-    </div>
+<div class="card ev-card-panel ev-challenge-card ev-card-hover">
+  <div class="card-image ev-card-image">
+    <img class="bg-img" [src]="challenge['image']">
+    <span class=" ev-card-title fs-14"><span><img [src]="challenge['image']"></span>{{challenge['title']}}</span>
   </div>
-  <div class="challenge-details">
-    <div class="challenge-title" (click)="challengeClicked()">
-      {{challenge['title']}}
-    </div>
-    <div class="challenge-host">
-    	Organized by: <span>{{challenge['creator']['team_name']}}</span>
-    </div>
-
-    <span class="challenge-type upcoming" *ngIf="isUpcoming">
-    	Upcoming
-    </span>
-    <span class="challenge-type ongoing" *ngIf="isOngoing">
-    	Ongoing
-    </span>
-    <span class="challenge-type past" *ngIf="isPast">
-    	Past
-    </span>
-
-    <div>
-    <div class="challenge-description">
-    	{{challenge['short_description']}}
-    </div>
-    </div>
-
-    <div>
-    	<div class="challenge-time-remaining">
-    		{{timeRemaining}}
-    	</div>
-    	<div class="challenge-time-range">
-    		<img src="assets/images/clock.png" /> {{startDate}}  -  {{endDate}}
-    	</div>
-    </div>
-    <div class="challenge-participate-div">
-    	<a class="btn btn-filter challenge-participate" (click)="participateInChallenge()">Participate</a>
-    </div>
-    <div class="challenge-labels">
-    	<span [class.is-starred]="stars['is_starred']" [class.is-clickable]="isLoggedIn" (click)="starToggle()" class="stars">
-        <i class="fas fa-star"></i>
-        {{stars['count']}}
-        </span>
-    	<div class="challenge-tags">
-    	<i class="fas fa-tag"></i>
-        <span *ngFor="let tag of tags">  &nbsp;{{tag}} &nbsp;|</span>
-    	</div>
+  <div class="card-content">
+    <p><strong class="text-light-black fs-12">Organized by</strong>
+      <br> <strong>{{challenge['creator']['team_name']}}</strong></p>
+    <p><strong class="text-light-black fs-12">Starts on</strong>
+      <br> <strong>{{challenge['start_date'] | date:'medium'}} {{challenge['start_zone']}}</strong></p>
+    <p><strong class="text-light-black fs-12">Ends on</strong>
+      <br> <strong>{{challenge['end_date'] | date:'medium'}} {{challenge['end_zone']}}</strong></p>
+    <div class="margin-top-btm" *ngIf="this.routerPublic.url === '/challenges/me'">
```

Comment:
Can you take `/challenges/me` from a constant variable in `.ts` file?

Reply:
done